include/gatekeeper_net.h

```diff
@@ -307,6 +307,11 @@ struct net_config {
 	 */
 	int                  back_iface_enabled;
 
+	/* Parameters to setup the mailbox instance. */
+	unsigned             mailbox_max_entries;
+	unsigned             mailbox_mem_cache_size;
+	unsigned             mailbox_burst_size;
```

Comment:
Add `int` to each type and also more comments to describe what each parameter does.